jar/jar_test.go

```diff
@@ -33,54 +34,181 @@ func TestParse(t *testing.T) {
 	testCases := []struct {
 		filename string
 		wantBad  bool
+		wantCVE  []cveID
```

Comment:
nit: wantCVEs given it can be more than one CVE?

Reply:
Done